src/gui/folderman.cpp

```diff
@@ -884,16 +885,12 @@ bool FolderMan::checkVfsAvailability(const QString &path, Vfs::Mode mode) const
     return unsupportedConfiguration(path) && Vfs::checkAvailability(path, mode);
 }
 
-Folder *FolderMan::addFolderFromWizard(const AccountStatePtr &accountStatePtr, const QString &localFolder, const QString &remotePath, const QUrl &webDavUrl, const QString &displayName, bool useVfs)
+Folder *FolderMan::addFolderFromWizard(const AccountStatePtr &accountStatePtr, FolderDefinition &&folderDefinition, bool useVfs)
```

Comment:
It was a micro optimisation to reduce the amount of copies.
The only way to go through would be to make FolderDefinition non copyable.
Do you want me to make it a const reference?

Reply:
I think it would be more idiomatic to use a const lvalue reference since you do not intend to write to the reference. Or does the rvalue reference have any advantage over a regular reference?

Reply:
I write on the folder definiton